packages/core/src/utils/createMap/setupFontawesome.ts

```diff
@@ -0,0 +1,26 @@
+import fontawesomeFree from '@fortawesome/fontawesome-free/css/all.css?inline'
```

Comment:
https://vitejs.dev/guide/features#disabling-css-injection-into-the-page

Since we're now programmatically handling where fontawesome styles are placed, automatic CSS injection has been removed. This is also nice to separate fontawesome styles (which require separate handling) from all the other styles (which can just *be* there in the shadows and work).